src/ResourceManager/Automation/Commands.Automation/Common/AutomationClientDSC.cs

```diff
@@ -819,7 +819,7 @@ public void DeleteDscNode(string resourceGroupName, string automationAccountName
                 azureVmResourceGroup = resourceGroupName;
             }
 
-            string deploymentName = System.DateTimeOffset.Now.LocalDateTime.ToString("yyyyMMddhhmmss");
+            string deploymentName = Guid.NewGuid().ToString();
 
```

Comment:
Any reason not to use  [implicit "type" var](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/var)?